code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -329,6 +330,8 @@ MATCHBOXES ARE ALSO IN FANCY.DM
 	if(ismob(loc))
 		var/mob/M = loc
 		M.update_inv_wear_mask(0)
+		if(ishuman(loc))
```

Comment:
There's no need for the check here (and if there was, you should be using M, not loc, for consistency). update_inv_ears is defined at the mob level.